cockroachdb/templates/job.init.yaml

```diff
@@ -122,16 +122,17 @@ spec:
                       --cluster-name={{.}} \
                       {{- end }}
                       --host={{ template "cockroachdb.fullname" . }}-0.{{ template "cockroachdb.fullname" . -}}
-                              :{{ .Values.service.ports.grpc.internal.port | int64 }}
+                              :{{ .Values.service.ports.grpc.internal.port | int64 }} \
                   2>&1);
 
                   local exitCode="$?";
                   echo $output;
 
-                  if [[ "$exitCode" == "0" || "$output" == *"cluster has already been initialized"* ]]
-                    then break;
+                  if [[ "$output" =~ .*"Cluster successfully initialized".* || "$output" =~ .*"cluster has already been initialized".* ]]; then
+                    break;
                   fi
 
+                  echo "Sleeping for 5 seconds"
```

Comment:
Can you add comments why we need to `sleep` here please.

Reply:
Added reason in the echo statement directly.